docs/source/generics.rst

```diff
@@ -39,8 +68,16 @@ generic class that represents a stack:
        def empty(self) -> bool:
            return not self.items
 
+.. note::
+
+    There are currently no plans to deprecate the legacy syntax.
+    You can freely mix code using the new and old syntax variants,
+    even within a single file.
```

Comment:
PEP 695 does say you can't mix them within the same *class* (https://peps.python.org/pep-0695/#compatibility-with-traditional-typevars). The error cases are quite obscure though, so I'd be OK with not mentioning that.